caas/kubernetes/provider/bootstrap.go

```diff
@@ -294,8 +329,25 @@ func (c controllerStack) Deploy() (err error) {
 	return nil
 }
 
+func (c controllerStack) getControllerSvcSpec(cloudType string) (*controllerServiceSpec, error) {
+	if spec, ok := controllerServiceSpecs[cloudType]; ok {
+		if spec.serviceType == "" {
+			// serviceType is required.
+			return nil, errors.NotValidf("serviceType is empty for %q", cloudType)
+		}
+		return spec, nil
+	}
+	return nil, errors.NotValidf("cloudType %q", cloudType)
```

Comment:
We should just use a default as there are lots of cloud types, eg joyent, rackspace etc

Reply:
ah, right, added a default svc spec for any other cloud is not listed here